src/utilities/dates.ts

```diff
@@ -1,11 +1,14 @@
 import { parse, isValid, isDate as dateFnsIsDate } from 'date-fns'
+import { localization } from '@/localization'
 import { secondsToApproximateString } from '@/utilities/seconds'
 import { dateFunctions, toDate, formatDateInTimezone } from '@/utilities/timezone'
 
 const dateTimeNumericFormat = 'yyyy/MM/dd hh:mm:ss a'
 const timeNumericFormat = 'hh:mm:ss a'
 const timeNumericShortFormat = 'hh:mm a'
 const dateFormat = 'MMM do, yyyy'
+const dateMonthDayFormat = 'MMM d'
+const dateMonthDayYearFormat = 'MMM d, yyyy'
```

Comment:
This is almost identical to the dateFormat. We should pick one and stick to it.

Reply:
Going off the designs for this one but it doesn't feel like dropping ordinality is a significant departure? Certainly not all dates should be written as "Feb 21st"

Reply:
The [style guide](https://www.notion.so/prefect/Formatting-Basics-1d4d83bcb2ba471ead90910aeb5913b2) defines our date formats. 

We deliberately picked two formats for dates/times. One "pretty" (dateFormat) and one more spreadsheet like for lists (dateNumeric). IMO introducing variations just introduces inconsistency and also makes devs/designers have to think about what format they should use. 

The style guide also has some rules for "relative" time though we haven't implemented that yet (I'd like to).